service/src/main/java/com/epam/rft/atsy/service/passwordchange/validation/impl/PasswordValidatorImpl.java

```diff
@@ -31,11 +30,13 @@ public PasswordValidatorImpl(PasswordUniqueRule passwordUniqueRule) {
     @Override
     public boolean validate(PasswordChangeDTO passwordChangeDTO) throws PasswordValidationException {
         for (PasswordValidationRule passwordValidationRule : passwordValidationRules) {
-            if(!passwordValidationRule.isValid(passwordChangeDTO)) {
-                throw new PasswordValidationException(passwordValidationRule.getErrorMessage());
+            if (!passwordValidationRule.isValid(passwordChangeDTO)) {
+                throw new PasswordValidationException(passwordValidationRule.getErrorMessageKey());
             }
         }
 
+        // Is this really necessary? At this point all rules have been satisfied therefore
```

Comment:
These kind of questions should be asked in the chat, so it is not missed.

But you are right, this can be a void method.